tox.ini

```diff
@@ -1,10 +1,10 @@
 [tox]
 envlist =
     # list of supported Django/Python versions:
-    # https://docs.djangoproject.com/en/3.1/faq/install/#what-python-version-can-i-use-with-django
-    py{36,37,38,39}-dj{22,31,32}
-    py{38,39}-dj{40}
-    py{38,39}-djmain
+    # https://docs.djangoproject.com/en/dev/faq/install/#what-python-version-can-i-use-with-django
+    py{36,37,38,39,310}-dj{22,31,32}
```

Comment:
3.10 won't be supported against Django 2.2/3.1. It also not _as yet_ officially supported by 3.2. (It's due to be included in the next release).